cmd/prometheus/main.go

```diff
@@ -262,6 +268,9 @@ func main() {
 	a.Flag("web.listen-address", "Address to listen on for UI, API, and telemetry.").
 		Default("0.0.0.0:9090").StringVar(&cfg.web.ListenAddress)
 
+	a.Flag("auto-gomemlimit.ratio", "The ratio of reserved GOMEMLIMIT memory to the detected maximum container or system memory").
```

Comment:
maybe we could also state here that it only takes effect when the feature `auto-gomemlimit` is enabled?